docs/src/pages/components/autocomplete/Playground.tsx

```diff
@@ -11,7 +11,7 @@ export default function Playground() {
   const flatProps = {
     options: top100Films.map(option => option.title),
   };
-  const [value, setValue] = React.useState(null);
+  const [value, setValue] = React.useState<FilmOptionType>();
```

Comment:
It needs to be null as the default value so the component is considered controlled.
```suggestion
  const [value, setValue] = React.useState<FilmOptionType | null>(null);
```